spec/spec_helper_local.rb

```diff
@@ -6,7 +6,7 @@
 RSpec.configure do |config|
   config.mock_with :rspec
 
-  config.before :each do
+  config.before do
```

Comment:
If this is a common problem, we should really solve it in https://github.com/voxpupuli/voxpupuli-test IMHO.